cpu/cc26x0/periph/rfc.c

```diff
@@ -131,6 +131,7 @@ bool rfc_ping_test(void)
     pingCommand.commandID = CMDR_CMDID_PING;
     RFC_DBELL->CMDR = (uint32_t) (&pingCommand);
     while (!RFC_DBELL->CMDSTA); /* wait for cmd execution */
+    printf("rfc_ping fails without a printf reading cmdsta %lu\n", RFC_DBELL->CMDSTA);
```

Comment:
note that the mere presence of this line affects the evaluation in the interrupt which happens before the printf is reached oO
can't wait to see if anyone finds the reason